roles/smallfile-bench/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
+- set_fact:
+    smallfile_path: /mnt/pvc
+  when: smallfile.storageclass is defined
+
+- set_fact:
+    smallfile_path: /tmp/test_smallfile/
+  when: smallfile.storageclass is not defined
+
+- name: Create PVC(s)
+  k8s:
+    definition:
+      kind: PersistentVolumeClaim
+      apiVersion: v1
+      metadata:
+        name: claim{{item}}
+        namespace: "{{ operator_namespace }}"
+        annotations:
+          volume.beta.kubernetes.io/storage-class: "{{smallfile.storageclass}}"
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: "{{smallfile.storagesize}}"
+  with_sequence: start=1 count={{smallfile.clients}}
+  when: smallfile.clients > 0 and smallfile.storageclass is defined and smallfile.storagesize is defined
+
+- name: Generate smallfile test input yaml
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        namespace: "{{ operator_namespace }}"
+        name: smallfile-test
+      data:
+        smallfilejob: "{{ lookup('template', 'smallfilejob.other_parameters.yaml.j2') }}"
+
+- name: Start Run(s) with PVC
+  k8s:
+    definition:
+      kind: Job
+      apiVersion: batch/v1
+      metadata:
+        name: "{{ meta.name }}-smallfile-client-{{ item }}-benchmark"
+        namespace: "{{ operator_namespace }}"
+      spec:
+        ttlSecondsAfterFinished: 600
+        template:
+          metadata:
+            labels:
+              app: smallfile-benchmark
+          spec:
+            containers:
+              - name: benchmark-server
+                image: "quay.io/nichawla/smallfile_container:testing"
+                command: ["/bin/sh", "-c"]
+                workingDir: /root/smallfile-master/
+                args:
+                  - arr=$(python <<< "print(' '.join({{smallfile.operation}}))");
+                    rm -rf {{smallfile_path}}/RESULTS ;
+                    mkdir {{smallfile_path}}/RESULTS;
+                    mkdir {{smallfile_path}}/smallfile_test_data;
```

Comment:
do you mean "mkdir -p {{smallfile_path}}/smallfile_test_data"  ?   Otherwise this will be an error, though because of the shell syntax you are using, the error will be ignored.

Reply:
will add -p in mkdir